orchestra/tests/helpers/fixtures.py

```diff
@@ -245,4 +251,52 @@ def setup_models(test_case):
                 task=task,
                 status=assignment_status,
                 assignment_counter=i,
-                in_progress_task_data=task_data)
+                in_progress_task_data=task_data,
+                start_datetime=parse(
+                    '2015-10-12T0{}:00:00+00:00'.format(2 + i)))
+
+
+def task_back_and_forth(test):
```

Comment:
in the next iteration, i'll change the name of this thing and add a docstring.  roughly: creates the back-and-forth assignment iteration and timing information so we can test latency reporting, etc.